src/test/java/formflow/library/upload/UploadJourneyTests.java

```diff
@@ -24,47 +24,58 @@ public void setUp() throws IOException {
 
   @Test
   void documentUploadFlow() {
-    Assertions.assertThat(testPage.getTitle()).isEqualTo("Upload Documents");
+    assertThat(testPage.getTitle()).isEqualTo("Upload Documents");
     // Test accepted file types
     // Extension list comes from application.yaml -- form-flow.uploads.accepted-file-types
     uploadFile("test-platypus.gif", "uploadTest");
-    Assertions.assertThat(testPage.findElementsByClass("text--error").get(0).getText())
+    assertThat(testPage.findElementsByClass("text--error").get(0).getText())
         .isEqualTo(
-            "We aren't able to upload this type of file. Please try another file that ends in one of the following: .jpeg, .fake, .heic, .tif, .tiff");
+            "We aren't able to upload this type of file. Please try another file that ends in one of the following: .jpeg, .fake, .heic, .tif, .tiff, .pdf");
     testPage.clickLink("remove");
-    Assertions.assertThat(testPage.findElementTextById("number-of-uploaded-files-uploadTest")).isEqualTo("0 files added");
+    assertThat(testPage.findElementTextById("number-of-uploaded-files-uploadTest")).isEqualTo("0 files added");
+
     // Upload a file that is too big and assert the correct error shows - max file size in test is 1MB
     long largeFilesize = 21000000L;
     driver.executeScript(
         "$('#document-upload-uploadTest').get(0).dropzone.addFile({name: 'testFile.pdf', size: "
             + largeFilesize + ", type: 'not-an-image'})");
     int maxFileSize = 17;
-    Assertions.assertThat(driver.findElement(By.className("text--error")).getText()).contains(
+    assertThat(driver.findElement(By.className("text--error")).getText()).contains(
         "This file is too large and cannot be uploaded (max size: " + maxFileSize + " MB)");
     testPage.clickLink("remove");
-    Assertions.assertThat(testPage.findElementTextById("number-of-uploaded-files-uploadTest")).isEqualTo("0 files added");
+    assertThat(testPage.findElementTextById("number-of-uploaded-files-uploadTest")).isEqualTo("0 files added");
+
+    // Upload a password-protected file and assert the correct error shows
+    uploadPasswordProtectedPdf("uploadTest");
+    assertThat(testPage.findElementsByClass("text--error").get(0).getText())
+        .isEqualTo(
+            "We are unable to process password-protected PDF files. Please remove the password and try again or try another PDF file.");
```

Comment:
Do you know if we are able to easily pull in the `messages-form-flow.properties` file here?  It will make future developer's lives easier if someone tweaks those messages.   They/we won't have to find all these satellite spots to change.

Reply:
If we can do this, maybe it's worth doing in a separate ticket.